can-compute-stream.js

```diff
@@ -1,26 +1,37 @@
 var compute = require('can-compute');
-var define = require('can-define');
 var Kefir = require('kefir');
 var makeArray = require("can-util/js/make-array/make-array");
-var isArray = require("can-util/js/is-array/is-array");
 var assign = require("can-util/js/assign/assign");
+var canEvent = require('can-event');
 
-var singleComputeToStream = function (compute) {
+
+var computeStream = {};
+
+/*
+ * Pipes the value of a compute into a stream
+ */
+computeStream.singleComputeToStream = function (compute) {
 	return Kefir.stream(function (emitter) {
 		var changeHandler = function (ev, newVal) {
 			emitter.emit(newVal);
 		};
 
-		emitter.emit(compute());
 		compute.on('change', changeHandler);
+		emitter.emit(compute());
 
 		return function () {
 			compute.off('change', changeHandler);
 		};
 	});
 };
 
-var computeStream = function () {
+
+/*
+ * Converts all arguments passed into a single stream and resolves the resulting
+ * streams into a single stream. Assumes all arguments are computes and last
+ * argument is optionally a function.
+ */
+computeStream.toStreamFromCompute = function () {
```

Comment:
does this need to take multiple arguments? 

You don't need to use `hasOwnProperty` for isComputed.  Just read `.isComputed`